lib/fragment/structured_text/span/em.ex

```diff
@@ -9,4 +9,6 @@ end
 defimpl Span, for: Span.Em do
   # TODO
   def serialize(_, _link_resolver), do: ""
+  def open_tag(_span), do: "<i>"
```

Comment:
I believe "em" tag would be more appropriate her, see [ruby implentation](https://github.com/prismicio/ruby-kit/blob/d05a6c47f17c3a754e08abd321a64b8ca70193e9/lib/prismic/fragments/structured_text.rb#L122)